client/src/utils/API.js

```diff
@@ -1,7 +1,6 @@
 import axios from "axios";
 
```

Comment:
This whole api util is scary. you guys just assume that the axios request is going to work.